core/driver/sources/src/main/scala/org/virtuslab/ideprobe/config/IdeProbeConfig.scala

```diff
@@ -15,6 +14,5 @@ case class IdeProbeConfig(
 
 object IdeProbeConfig extends ConfigFormat {
   implicit val format: ConfigReader[IdeProbeConfig] = exportReader[IdeProbeConfig].instance
-  override implicit def hint[A] = ProductHint[A](ConfigFieldMapping(CamelCase, KebabCase))
```

Comment:
Hmmm okay so the trade-off has been finally resolved in favor of camelCase => **compatibility** rather than kebab-case => **conventionality**?

Reply:
yes